ntoskrnl/kdbg/i386/i386-dis.c

```diff
@@ -66,7 +66,12 @@ static int
 KdbpReadMemory(unsigned int Addr, unsigned char* Data, unsigned int Length,
 	       struct disassemble_info * Ignored)
 {
+#ifdef _WIN64
+  // FIXME: Addr type should be fixed, on all functions...
+  return KdbpSafeReadMemory(Data, (void *)(unsigned long long)Addr, Length); /* 0 means no error */
```

Comment:
Don't do this. Fix the parameter to be ULONG_PTR instead. Same for KdbpPrintAddressInCode.

Reply:
Superseded as part of 253362509ec64a2def87fdc818f3d553f23a20c6.